velox/functions/lib/DateTimeFormatter.h

```diff
@@ -191,5 +200,58 @@ std::shared_ptr<DateTimeFormatter> buildMysqlDateTimeFormatter(
 
 std::shared_ptr<DateTimeFormatter> buildJodaDateTimeFormatter(
     const std::string_view& format);
+namespace {
```

Comment:
The name of this header file is very specific: DateTimeFormatter. The new functions added to it do no have anything to do with formatting, hence, do not belong here. Let's move them into a separate .h file.

Reply:
thanks, I moved this logic into TimeUtil.h.